pkgs/top-level/python-packages.nix

```
@@ -3978,6 +3978,8 @@ in {
 
   kerberos = callPackage ../development/python-modules/kerberos { };
 
+  kerbroute = callPackage ../tools/security/kerbrute { };
```

Should go to `development/python-modules/kerbrute`

```suggestion
  kerbroute = callPackage ../development/python-modules/kerbrute { };
```

and added to `all-packages.nix` with `toPythonApplication`.

---

my bad, it shouldn't be `python-packages.nix` because it does not have any importable module. so callPackage directly in `all-packages.nix` .